docs/content/chunker.md

```diff
@@ -0,0 +1,237 @@
+Files larger than chunk_size will be split in chunks. By default 2 Gb.
+Enter a size with suffix k,M,G,T. Press Enter for the default ("2G").
+chunk_size> 1G
+Edit advanced config? (y/n)
+y) Yes
+n) No
+y/n> n
+Remote config
+--------------------
+[overlay]
+type = chunker
+remote = TestLocal:
+chunk_size = 2G
+--------------------
+y) Yes this is OK
+e) Edit this remote
+d) Delete this remote
+y/e/d> y
+```
+
+## Specifying the remote ##
+
+In normal use, make sure the remote has a `:` in. If you specify the
+remote without a `:` then rclone will use a local directory of that
+name.  So if you use a remote of `/path/to/secret/files` then rclone
+will encrypt stuff to that directory.  If you use a remote of `name`
+then rclone will put files in a directory called `name` in the current
+directory.
+
+#### How chunking works
+
+When rclone starts a file upload, chunker checks file size.
+If the file is less than configured chunk size, chunker simply passes
+it to the underlying remote. If the file is large chunker will
+transparently create temporary and stream them on the fly one by one.
+Each chunk will contain specified number of data byts, except for the
+last one which may have less data. If file size is unknown in advance,
+chunker will transparently create a temporary copy, record its size
+and repeat the above process. When upload is complete, temporary chunk
+files are given their final names.
+
+By default chunks names are `BIG_FILE_NAME.rclone-chunk.001`,
+`BIG_FILE_NAME.rclone-chunk.002` etc, because the default chunk name
+format is `*.rclone-chunk.###`. You can configure another name format
+using the `--chunker-name-format` option. The format uses asterisk
+`*` as a placeholder for the base file name and one or more consecutive
+hash characters `#` as a placeholder for the chunk number. There must be
+one and only one asterisk. The number of consecutive hashes defines the
+minimum left of a chunk number. If a chunk number has less digits than
+the number of hashes, it is left-padded by zeros. If there are more
+digits in the number, they are left as is. The numbering by default
+start from 1 but there is another option that allows user to start from 0.
+
+For example, if name format is `big_*-##.part`, and original file was
+named `data.txt` and numbering starts from 0, then the first chunk will be
+named `big_data.txt-00.part`, the 99th chunk will be `big_data.txt-98.part`
+and the 302nd chunk will be `big_data.txt-301.part`.
+
+Possible chunk files are ignored if their name does not match given format.
+
+#### Metadata
+
+By default chunker creates metadata object besides data chunks. It is named
+after the original file. Currenly then only supported format is `simplejson`,
+which keeps the following fields:
+
+- Name         - name of the composite file (equal to the meta file name)
+- Size         - total size of all chunks
+- PublicKey    - currently not used, always "null"
+- CreationDate - last modification time
```

Comment:
Perhaps this should be called `ModificationDate` if that is what it is?

Reply:
The format is obviously poor. It roots from https://github.com/yar229/WebDavMailRuCloud origins. My humble problem is a few terabytes of data kept using it :(
I propose to freeze this meta-format (as "wdmrcompat" or "yar229chunks" or "poormanjson", or choose a better name) and proceed to improving "simplejson", eg: remove PublicKey, replace CreationDate by ModificationTime, add HashSum+HashType. ok?

Reply:
Oh, I see... Well it isn't terrible and if it is an existing "standard" then OK!  Your transition plan sounds fine to me, but we should try to finalise it before we get to the v1.50 release.